lib/internal/Magento/Framework/View/Element/Template/File/Validator.php

```diff
@@ -105,16 +105,48 @@ public function isValid($filename)
     {
         $filename = str_replace('\\', '/', $filename);
         if (!isset($this->_templatesValidationResults[$filename])) {
-            $this->_templatesValidationResults[$filename] =
-                ($this->isPathInDirectories($filename, $this->_compiledDir)
-                    || $this->isPathInDirectories($filename, $this->moduleDirs)
-                    || $this->isPathInDirectories($filename, $this->_themesDir)
-                    || $this->_isAllowSymlinks)
-                && $this->getRootDirectory()->isFile($this->getRootDirectory()->getRelativePath($filename));
+            /**
```

Comment:
This should be removed.

Reply:
Hi @miguelbalparda.

What code should be removed exactly?

Reply:
Sorry about that, lines 108 to 113.

Reply:
@miguelbalparda I have removed comment and committed.